lib/fake_ifdh.py

```diff
@@ -75,10 +79,14 @@ def getRole(role_override: Optional[str] = None, verbose: int = 0) -> str:
             return role
 
     # if there's a role in the wlcg.groups of the token, pick that
-    if os.environ.get("BEARER_TOKEN_FILE", False):
-        with os.popen("decode_token.sh $BEARER_TOKEN_FILE", "r") as f:
-            token_s = f.read()
-            token = json.loads(token_s)
+    if os.environ.get("BEARER_TOKEN_FILE", False) and os.path.exists(
+        os.environ["BEARER_TOKEN_FILE"]
+    ):
+        with open(os.environ["BEARER_TOKEN_FILE"]) as f:
+            token_encoded = f.read(4096).strip()
```

Comment:
Why limit to 4096 bytes? That's only 2-3x what a current analysis token is, and we may see that explode (cf. recent request from mu2e). If you must have a limit, at least give us a couple orders of magnitude buffer.

Reply:
I was getting an error earlier on (because I wasn't stripping the newline off the file contents) and I thought I was perhaps getting a short read; so the 4096 was an attempt to be doubly sure it was reading the whole file... I should take that back out...